tests/integration_tests/test_gbm.py

```diff
@@ -0,0 +1,294 @@
+import contextlib
+import os
+import tempfile
+
+import pytest
+
+try:
+    import ray as _ray
+except ImportError:
+    _ray = None
+from marshmallow import ValidationError
+
+from ludwig.api import LudwigModel
+from ludwig.constants import MODEL_TYPE, TRAINER
+from tests.integration_tests.utils import binary_feature, category_feature, generate_data, number_feature, text_feature
+
+
+@contextlib.contextmanager
+def ray_start(num_cpus=3, num_gpus=None):
+    res = _ray.init(
+        num_cpus=num_cpus,
+        num_gpus=num_gpus,
+        include_dashboard=False,
+        object_store_memory=150 * 1024 * 1024,
+    )
+    try:
+        yield res
+    finally:
+        _ray.shutdown()
+
+
+@pytest.fixture(scope="module")
+def local_backend():
+    return {"type": "local"}
+
+
+@pytest.fixture(scope="module")
+def ray_backend():
+    num_workers = 2
+    num_cpus_per_worker = 1
+    return {
+        "type": "ray",
+        "processor": {
+            "parallelism": num_cpus_per_worker * num_workers,
+        },
+        "trainer": {
+            "use_gpu": False,
+            "num_workers": num_workers,
+            "resources_per_worker": {
+                "CPU": num_cpus_per_worker,
+                "GPU": 0,
+            },
+        },
+    }
+
+
+def run_test_gbm_output_not_supported(backend_config):
+    """Test that an error is raised when the output feature is not supported by the model."""
+    input_features = [number_feature(), category_feature(reduce_output="sum")]
+    output_features = [text_feature()]
+
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
tmpdir is a built-in pytest fixture.

Reply:
I was actually using the fixture before, but ran into issues so I switched to this style in accordance with test_ray.py